bcos-executor/src/executive/BillingTransactionExecutive.cpp

```diff
@@ -0,0 +1,76 @@
+#include "BillingTransactionExecutive.h"
+#include <bcos-codec/bcos-codec/wrapper/CodecWrapper.h>
+#include <bcos-framework/bcos-framework/executor/PrecompiledTypeDef.h>
+
+#ifdef WITH_WASM
+#include "../vm/gas_meter/GasInjector.h"
+#endif
+
+using namespace bcos::executor;
+using namespace bcos::codec;
+using namespace bcos::precompiled;
+
+CallParameters::UniquePtr BillingTransactionExecutive::start(CallParameters::UniquePtr input)
+{
+    uint64_t currentSeq = input->seq;
+    auto message = TransactionExecutive::execute(std::move(input));
+
+    if (currentSeq == 0)
+    {
+        CallParameters::UniquePtr callParam4AccountPre =
+            std::make_unique<CallParameters>(CallParameters::MESSAGE);
+        auto codec = CodecWrapper(m_blockContext.hashHandler(), m_blockContext.isWasm());
+        bytes getBalanceIn = codec.encodeWithSig("getAccountBalance()");
+        callParam4AccountPre->data = getBalanceIn;
+        callParam4AccountPre->senderAddress = input->senderAddress;
+        callParam4AccountPre->receiveAddress = ACCOUNT_ADDRESS;
+        auto balanceRet = callPrecompiled(std::move(callParam4AccountPre));
+        if(balanceRet->type == CallParameters::REVERT)
+        {
+            message->type = balanceRet->type;
+            message->status = balanceRet->status;
+            message->message = balanceRet->message;
+            return message;
+        }
+        s256 balance;
```

Comment:
balance和gas，都调整为用u256（Precompiled那边也在改成u256）

```suggestion
        u256 balance;
```